.github/workflows/integration.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       # Prerequisites
       - uses: ./.github/actions/restore-node
         with:
-          node-version: 18.18
+          version-identifier: 18.18
```

Comment:
We should revisit later, but I definitely want to understand why this one is using `18.18` currently.